File: insertion_sort.js
const insertion_sort = (array) => {
    const size = array.length - 1
    let i = 0
    while (i < size) {
        const next_element = array[i + 1]
        let k = i
        while (k >= 0 && array[k] > next_element) {
            array[k + 1] = array[k]
            k--
        }
        array[k + 1] = next_element
        i++
    }
    return array
}

console.log(insertion_sort([6, 5, 3, 2, 1100, 23, 45, 64, 76, 86, 3, 4, 5]))

File: insertion_sort.py
def insertion_sort(array):
    i = 0
    while i<len(array)-1:
        next_element = array[i+1]
        k = i
        while k>=0 and next_element<array[k]:
            array[k+1] = array[k]
            k-=1          
        array[k+1]=next_element
        i+=1
    return array

print(insertion_sort([123,32,23,54,654,876987,543,23]))